doc/developer/platform/ux.md

```diff
@@ -194,7 +194,7 @@ Terraform provider to manage these resources types.
 
 ### Admin API
 
-The admin API is a REST API hosted at <https://admin.cloud.materialize.com>.
+The admin API is a REST API hosted at <https://admin.console.materialize.com>.
```

Comment:
The admin API is still hosted at admin.cloud.materialize.com.

Reply:
Thanks!